onnx/defs/nn/defs.cc

```diff
@@ -959,19 +962,21 @@ ONNX_OPERATOR_SET_SCHEMA(
           auto w_type = ctx.getInputType(3);
           if (nullptr == x_type || nullptr == w_type || x_type->value_case() != TypeProto::kTensorType ||
               w_type->value_case() != TypeProto::kTensorType) {
-            fail_type_inference("inputs are expected to have tensor type.");
+            fail_type_inference("inputs are expected to have tensor type in ", ctx.getDisplayName(), ".");
```

Comment:
[…] not [actual type]

Reply:
The type is int. If we have a function which returns the type name, why not using it. Otherwise I prefer to let the message the way it is. All the error messages must be improved, not only this one.

Reply:
Another PR is necessary.